drivers/CoreLED/include/CoreLED.h

```diff
@@ -32,6 +34,8 @@ class CoreLED : public interface::LED
 		static constexpr auto reset = std::to_array<uint8_t>({0x00, 0x00, 0x00, 0x00});
 		static constexpr auto end	= std::to_array<uint8_t>({0x00, 0x00});
 	};
+
+	void showColor(RGB color);
```

Comment:
pour éviter les confusions avec `showColor(void)` (cf commentaire de Yann), je propose:

```suggestion
private: 
	void sendAndDisplay(RGB color);
```